Playbooks/AS-PagerDuty-Integration/README.md

```diff
@@ -0,0 +1,136 @@
+#
+### Requirements
+                                                                                                                                     
+The following items are required under the template settings during deployment: 
+
+* A PagerDuty integration key
+                                                                                                                                    
+# 
+### Setup
+                                                                                                                                     
+To Create a PagerDuty Integration Key:
+ 
+Log into your PagerDuty account and under the "**Services**" menu option, select "**Service Directory**". From the page, click the "**New Service**" button.
+ 
+![PagerDuty_Create_Integration_Key_1](Images/PagerDuty_Create_Integration_Key_1.png)
+
+                                                                                                       
+Add a Name and optional Description like the following:
+
+![PagerDuty_Create_Integration_Key_2](Images/PagerDuty_Create_Integration_Key_2.png)              
+                                                                                       
+                                                                                         
+Select an escalation policy for your service.
+
+![PagerDuty_Create_Integration_Key_3](Images/PagerDuty_Create_Integration_Key_3.png)
+
+                                                                                           
+Finally, select the Microsoft Azure option under the Integrations section, then click "**Create Service**".
+
+![PagerDuty_Create_Integration_Key_4](Images/PagerDuty_Create_Integration_Key_4.png)
+
+                                                                                                  
+You will be redirected to the service you have created. Under the Integrations tab view, you will find your integration key. This will be needed for the deployment of this playbook.
+                                                                                                  
+![PagerDuty_Create_Integration_Key_5](Images/PagerDuty_Create_Integration_Key_5.png)
+                                                                                           
+                                                                                                 
+#
+### Deployment                                                                                                         
+                                                                                                        
+To configure and deploy this playbook:
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-PagerDuty-Integration%2Fmain%2Fazuredeploy.json" target="_blank">
```

Comment:
Incorrect link added for "Deploy to Azure". azuredeploy.json is not present in the main folder.
https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-PagerDuty-Integration%2Fmain%2Fazuredeploy.json

Deploy to Azure button is added twice to the README.md. Ensure this is present only once, preferable where steps for deployment are mentioned.

Reply:
This is resolved